packages/code-studio/src/dashboard/DashboardContainer.jsx

```diff
@@ -230,7 +230,7 @@ export class DashboardContainer extends Component {
     setDashboardLinks(id, [...links]);
   }
 
-  initLayout() {
+  async initLayout() {
```

Comment:
gwhaaa..... @mofojed I think I inherited this from you... unsure what I should do w/ it.

Reply:
Get rid of it, if you rebase off the latest it should already have this.

Reply:
done, tyvm.